phpseclib/File/X509.php

```diff
@@ -3117,12 +3118,13 @@ public function removeExtension($id)
      *
      * @param string $id
      * @param array $cert optional
+     * @param string $path
      * @access public
      * @return mixed
      */
-    public function getExtension($id, $cert = null)
+    public function getExtension($id, $cert = null, $path)
     {
-        return $this->getExtensionHelper($id, $cert);
+        return $this->getExtensionHelper($id, $cert, $path);
```

Comment:
Looks like a unit test will need to be written for this too. This is only called with a third parameter by `getRevokedCertificateExtension`.

`getExtensions` (plural vs singular) probably ought to be updated similarly, as well, and then `getRevokedCertificateExtensions` updated to call `getExtensions` instead of `getExtensionsHelper`.